deployment/crawler-legacy-cronjob.yaml

```diff
@@ -0,0 +1,58 @@
+---
```

Comment:
That is probably ok. For the compose validation, In .github/workflows/validate-pr.yaml there is a config.yaml with a fake config to crawl a github organization. https://github.com/change-metrics/monocle/blob/master/.github/workflows/validate-pr.yaml#L65-L89 this way the crawler-legacy keeps running.

Reply:
then we should migrate the config https://github.com/change-metrics/monocle/blob/master/deployment/monocle-config.yaml to be like what you sent?

Reply:
Anyway let's not bother with this the service well be remove sooner than later.

Reply:
So let's remove it when we obsolete it totally.. as long as we are still using it this should stay (we do run this currently)